swap_meet/clothing.py

```diff
@@ -1,2 +1,7 @@
-class Clothing:
-    pass
+from .item import Item 
+class Clothing(Item):
+    def __init__(self, condition = None, category = "Clothing"):
+        super().__init__(category, condition)
```

Comment:
We don't really want to let users of our  class change the category for a Clothing instance to something else (at least not from the constructor). We can leave the category out of the constructor parameters entirely, and pass the desired category literally in to the parent's constructor as follows:
```python
    def __init__(self, condition=None):
        super().__init__(category="Clothing", condition=condition)
```